packages/core/src/stryker-cli.ts

```diff
@@ -64,12 +64,12 @@ export class StrykerCli {
       })
       .option(
         '-f, --files <allFiles>',
-        '[DEPRECATED, please use `--ignorePatterns` instead] A comma separated list of patterns used for selecting all files needed to run the tests. For a more detailed way of selecting input files, please use a configFile. Example: src/**/*.js,!src/index.js,a.js,test/**/*.js.',
+        '[DEPRECATED, please use the inverse option `--ignorePatterns` instead] A comma separated list of patterns used for selecting all files needed to run the tests. For a more detailed way of selecting input files, please use a configFile. Example: src/**/*.js,!src/index.js,a.js,test/**/*.js.',
         list
       )
       .option(
         '--ignorePatterns <filesToIgnore>',
-        'A comma separated list of patterns used for specifying which files need to be ignored. Example: --ignorePatterns dist. Note that `node_modules`, `.git` and others are always ignored. Note: this cannot be combined with "files".',
+        'A comma separated list of patterns used for specifying which files need to be ignored. Example: --ignorePatterns dist. Should only be used in cases where for example large files slow down stryker. Note that `node_modules`, `.git` and others are always ignored. Note: this cannot be combined with "files".',
```

Comment:
```suggestion
        'A comma separated list of patterns used for specifying which files need to be ignored. This should only be used in cases where you experience a slow Stryker startup, because too many (or too large) files are copied to the sandbox that are not needed to run the tests. For example, image or movie directories. Note that `node_modules`, `.git` and others are always ignored. Note: this cannot be combined with "files".  Example: --ignorePatterns dist',
```